Jenkinsfile

```diff
@@ -813,30 +813,30 @@ pipeline {
                         buildHipClangJobAndReboot( setup_flags: Tensile_setup + Full_test + extra_log_env + Int8_flags, build_env: Tensile_build_env + extra_log_env, test_flags: ' --verbose ', gpu_arch: "gfx908:xnack-", miotensile_version: Tensile_version, target_id: "ON")
                     }
                 }
-                // stage('Fp32 Hip Tensile-Latest All gfx90a') {
-                //     agent{ label rocmnode("gfx90a") }
-                //     steps{
-                //         buildHipClangJobAndReboot( setup_flags: Tensile_setup + Full_test + extra_log_env, build_env: Tensile_build_env + extra_log_env, test_flags: ' --verbose ', gpu_arch: "gfx90a:xnack-", miotensile_version: Tensile_version, target_id: "ON")
-                //     }
-                // }
-                // stage('Fp16 Hip Tensile-Latest All gfx90a') {
-                //     agent{ label rocmnode("gfx90a") }
-                //     steps{
-                //         buildHipClangJobAndReboot( setup_flags: Tensile_setup + Full_test + extra_log_env + Fp16_flags, build_env: Tensile_build_env + extra_log_env, test_flags: ' --verbose ', gpu_arch: "gfx90a:xnack-", miotensile_version: Tensile_version, target_id: "ON")
-                //     }
-                // }
-                // stage('Bf16 Hip Tensile-Latest All gfx90a') {
-                //     agent{ label rocmnode("gfx90a") }
-                //     steps{
-                //         buildHipClangJobAndReboot( setup_flags: Tensile_setup + Full_test + extra_log_env + Bf16_flags, build_env: Tensile_build_env + extra_log_env, test_flags: ' --verbose ', gpu_arch: "gfx90a:xnack-", miotensile_version: Tensile_version, target_id: "ON")
-                //     }
-                // }
-                // stage('Int8 Hip Tensile-Latest All gfx90a') {
-                //     agent{ label rocmnode("gfx90a") }
-                //     steps{
-                //         buildHipClangJobAndReboot( setup_flags: Tensile_setup + Full_test + extra_log_env + Int8_flags, build_env: Tensile_build_env + extra_log_env, test_flags: ' --verbose ', gpu_arch: "gfx90a:xnack-", miotensile_version: Tensile_version, target_id: "ON")
-                //     }
-                // }
+                stage('Fp32 Hip Tensile-Latest All gfx90a') {
```

Comment:
@ce1adon I am not sure if this is the way to enable these tests.
May I ask how is **params.MIOPENTENSILE_LATEST** being evaluated? Usually these tests are bypassed anyway, right?

Reply:
Hey Jun, yes, this commit enables all the commented tests in miopentensile latest stage. The test is then manually enabled in Jenkins. Since the Jenkinsfile changed, I included all the regular tests to run to make sure no one is affected by this stage. I will revert your latest commit.

Reply:
My bad. I was trying to save some CI resources since all other stages have passed before, now we only need to test the MIOPENTENSILE_LATEST stages. Please make corrections as you may see fit. Thanks!

Reply:
I see. Will disable other stages.

Reply:
@junliume 
> May I ask how is params.MIOPENTENSILE_LATEST being evaluated?

It is evaluated at line 763.